examples/podman/stepca.container.md

```diff
@@ -0,0 +1,45 @@
+## Example [Podman Quadlet container](https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#container-units-container) file
+
+* `~/.config/containers/systemd/stepca.container` (rootless)
+* `/etc/containers/systemd/stepca.container`      (rootful)
+
+```
+[Unit]
+Description=Smallstep Certificate Authority
+After=network-online.target
+
+[Container]
+PodmanArgs=--memory 25m --cpus 0.20
+PidsLimit=60
+DropCapability=ALL
+AutoUpdate=registry
+ContainerName=stepca
+DropCapability=ALL
+Environment=TZ="Europe/London"
+Environment="DOCKER_STEPCA_INIT_NAME=Example CA"
+Environment=DOCKER_STEPCA_INIT_DNS_NAMES=ca.custom.domain,10.89.0.10,localhost,127.0.0.1
+Environment=DOCKER_STEPCA_INIT_PROVISIONER_NAME=admin@custom.domain
+Environment=DOCKER_STEPCA_INIT_SSH=true
+Environment=DOCKER_STEPCA_INIT_ACME=true
+Environment=DOCKER_STEPCA_INIT_PASSWORD_FILE=/run/secrets/stepca
+HostName=stepca
+PodmanArgs=--privileged
+# Alpine image
```

Comment:
Let's just remove the Alpine and TPM container example comments. If a user is going to customize what image they are going to use they will do so and the examples we are giving don't give any context on why they would use something different. 

For example, if they want to use a TPM they need to pass in the TPM device as well and also configure step ca to use it. Providing an alternative to the container image here could lead to confusion and/or a broken setup.